README.md

```diff
@@ -161,4 +164,4 @@ I would like to thank Microverse for the opportunity to dream big.
 This project is [GNsU](./LICENSE) licensed.
```

Comment:
- [ ] Kindly add the MIT license for your project this will make your readme more professional, you can do it here on [Github](https://github.com/sawmon71293/BookStore/community/license/new?branch=develop&filename=LICENSE&template=mit), please add this as it was requested by the previous reviewer 👍🏿

Reply:
@prolajumokeoni , 
I've added GNU license file and the link is not broken now. Why MIT ? They have any difference.

Reply:
I guess they are optional right?? You can either use of these or only MIT??